Resources/Prototypes/_NF/Catalog/VendingMachines/Inventories/lesslethalvend.yml

```diff
@@ -0,0 +1,23 @@
+- type: vendingMachineInventory
+  id: LessLethalVendInventory
+  startingInventory:
+    WeaponDisabler: 10
+    WeaponPistolMk58Nonlethal: 20
```

Comment:
Add in the mosin and the double barrel bean bag filled as well as the beanbag shotgun shells